flood_api/settings.py

```diff
@@ -7,15 +7,9 @@ class Settings(BaseSettings):
     uvicorn_host: str = "0.0.0.0"
     uvicorn_reload: bool = True
     uvicorn_proxy_headers: bool = False
-    detailed_data_path: str = (
-        "s3://databricks-data-openepi/glofas/processed/newest/processed_detailed_forecast.parquet/"
-    )
-    summary_data_path: str = (
-        "s3://databricks-data-openepi/glofas/processed/newest/processed_summary_forecast.parquet/"
-    )
-    threshold_data_path: str = (
-        "s3://databricks-data-openepi/glofas/auxiliary-data/processed_thresholds.parquet/"
-    )
+    detailed_data_path: str = "s3://databricks-data-openepi/glofas/processed/newest/processed_detailed_forecast.parquet/"
+    summary_data_path: str = "s3://databricks-data-openepi/glofas/processed/newest/processed_summary_forecast.parquet/"
+    threshold_data_path: str = "s3://databricks-data-openepi/glofas/auxiliary-data/processed_thresholds.parquet/"
```

Comment:
The Black formatter in my Vscode insisted on parentheses around these strings, which the linter in the CI wasn't too happy with. Which one is correct?